src/contexts/GlobalContext.jsx

```diff
@@ -0,0 +1,542 @@
+import { flushSync } from "react-dom";
```

Comment:
We have to add the Copyright at the top of the files as per Guidelines

Reply:
Updated with copyrights